.travis.yml

```diff
@@ -1,9 +1,9 @@
 sudo: required
 env:
   matrix:
-    - PLATFORM=centos DISTRO=el OS_VERSION=7 DOCKER_IMAGE=centos:centos7
-    - PLATFORM=debian DISTRO=debian OS_VERSION=jessie DOCKER_IMAGE=debian:jessie
-    - PLATFORM=ubuntu DISTRO=ubuntu OS_VERSION=trusty DOCKER_IMAGE=ubuntu:trusty
+    - PLATFORM=centos DISTRO=el OS_VERSION=7 DOCKER_IMAGE=centos:centos7 LABEL=7
```

Comment:
LABEL is a duplicate of OS_VERSION which is already exported as an environment variable here.